index.html

```diff
@@ -0,0 +1,192 @@
+
+    <style>
+        body {
+            background-color: #ffeead;
+			font-size: 1.2em;
+            font-weight: bold;
+        }
+        .container {
+            display: grid;
+            grid-gap: 3px;
+            grid-template-columns: repeat(12, 1fr);
+            grid-template-rows: 50px auto 50px;
+            height: 100%;
+        }
+        header {
+            grid-column: 2 / -1;
+            background-color: #fd726d;
+        }
+
+        nav {
+            display: flex;
+            align-items: center;
+            padding: 14px;
+            color: #ffeead;
+        }
+
+		.center_text {
+            color: #ffeead;
+            display: flex;
+            align-items: center; 
+            justify-content: center; 
+        }
+
+		nav > div:not(:last-child) { 
+            margin-right: 20px;
+        }
+       
+        nav > div:last-child {
+			margin-left: auto;
+        }
+
+        .logo {
+            grid-row: 1 / 2;
+            background-color: #98cdb5;
+        }
+
+        sidebar {
+            grid-row: 2 / 3;
+            background-color: #20cb9b;
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            padding: 30px;
+        }
+
+        sidebar div:not(:first-child) {
+            margin-top: 20px;
+        }
+
+        sidebar > div:first-child {
+            margin-bottom: auto;
+        }
+
+        .content {
+            grid-column: 2 / -1;
+            background-color: #fdcb65;
+            display: grid;
+            padding: 20px;
+            grid-gap: 8px;
+            grid-template-columns: repeat(12, 1fr);
+            grid-template-rows: auto auto;
+        }
+
+        .content1 {
+            background-color: #d6ccd3;
+            grid-column: 1 / 4;
+        }
+        .content2 {
+            background-color: #eaa1a7;
+            grid-column: 4 / 9;
+        }
+        .content3 {
+            background-color: #bdcebf;
+            grid-column: 1 / 9;
+        }
+        .content4 {
+            background-color: #ada399;
+            grid-column: 9 / -1;
+            grid-row: 1 / 3;
+        }
+
+        footer {
+            grid-column: 1 / -1;
+            background-color: #dea0de;
+        }
+    
+
+        @media only screen and (max-width: 600px) {
+
+        .content > div {
+            grid-column: 1 / -1;	
+        }
+
+        .content4 {
+            grid-row: 4 / 5;
+        }
+
+        header {
+            grid-column: 1 / -1;
+        }
+
+        nav {
+            flex-wrap: wrap;
+        }
+
+        nav > div {
+            display: flex;
+            flex: 1 1 100%;
+            justify-content: center; 
+        }
+
+        nav > div:not(:last-child) { 
+            margin-right: 0px;
+        }
+
+        .container {
+            display: grid;
+            grid-gap: 3px;
+            grid-template-columns: repeat(12, 1fr);
+            grid-template-rows: 50px auto auto auto 50px;
```

Comment:
For the .content row you could set `1fr` instead of `auto`. In this case you will have more appropriate heights of all blocks